ui/tests/integration/components/auth-jwt-test.js

```diff
@@ -86,6 +88,7 @@ module('Integration | Component | auth jwt', function(hooks) {
   setupRenderingTest(hooks);
 
   hooks.beforeEach(function() {
+    this.openSpy = sinon.spy(fakeWindow.proto(), 'open');
```

Comment:
ahh, so are we spying on the class `fakeWindow` instead of an instance of `fakeWindow`?

Reply:
Yep that was the big change here, spying on the prototype instead of the instance. Because of the way JS object model works, calling the method looks it up on the instance and if it's not there looks it up on the prototype and calls that. After updating the various bits, that's what 9 of the tests were failing on - `open` wasn't getting called, so I tried it this way and it does.